pg_bloat_check.py

```diff
@@ -565,21 +565,35 @@ def rebuild_index(conn, index_list):
 
 def convert_to_bytes(val):
 
+   # immediately return if val is a number
+   # (i.e., no units were provided)
+   if val.isdigit():
+       return val
+
    # split numbers from unit descriptor
+   # we assume format is "####kb" or "#### MB" etc.
```

Comment:
Just for consistency, can the example here not have a space since the following line is saying it assumes there are no spaces?

Reply:
Silly me 🤦🏻